creator-kit-1-cascoda.config

```diff
@@ -3,6 +3,7 @@ CONFIG_TARGET_pistachio_marduk=y
 CONFIG_TARGET_pistachio_marduk_marduk_ca8210=y
 CONFIG_PACKAGE_button-led-controller=y
 CONFIG_IMAGEOPT=y
+CONFIG_LIBCURL_OPENSSL=y
```

Comment:
Dependency not met correctly.  CONFIG_PACKAGE_libcurl needed to be enabled first